commons/src/main/java/info/metadude/android/eventfahrplan/commons/temporal/DateParser.kt

```diff
@@ -0,0 +1,39 @@
+package info.metadude.android.eventfahrplan.commons.temporal
+
+import org.threeten.bp.Instant
+import org.threeten.bp.ZoneOffset
+import org.threeten.bp.ZonedDateTime
+import org.threeten.bp.format.DateTimeFormatter
+import org.threeten.bp.format.DateTimeParseException
+
+object DateParser {
+
+    /**
+     * Parses the given [text] and returns its date value represented in milliseconds.
+     *
+     * Expects the [text] to be provided in [ISO_DATE_TIME][DateTimeFormatter.ISO_DATE_TIME]
+     * format (e.g. 2019-01-01T00:00:00Z).
+     */
+    fun parseDateTime(text: String): Long {
```

Comment:
I see that info.metadude.android.eventfahrplan.network.temporal.DateParser has very similar functionality. Why not extend / reuse it?